common/static/css/edx-cc.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: 'edx-cc';
+  src: url('../fonts/edx-cc/edx-cc.eot?52318265');
+  src: url('../fonts/edx-cc/edx-cc.eot?52318265#iefix') format('embedded-opentype'),
+       url('../fonts/edx-cc/edx-cc.woff?52318265') format('woff'),
+       url('../fonts/edx-cc/edx-cc.ttf?52318265') format('truetype'),
+       url('../fonts/edx-cc/edx-cc.svg?52318265#edx-cc') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+/* Chrome hack: SVG is rendered more smooth in Windozze. 100% magic, uncomment if you need it. */
+/* Note, that will break hinting! In other OS-es font will be not as sharp as it could be */
```

Comment:
@sarina I modified the paths of the original edx-cc font when I added it to this PR. However, I must have missed the reference to the .svg version, it shows as a comment in my editor. I updated the reference in commit: 6175e3134066a9f1e0939f9d31a3d8c735192255.